src/github.com/matrix-org/go-neb/services/slackapi/message.go

```diff
@@ -84,6 +85,7 @@ var netClient = &http.Client{
 	Timeout: time.Second * 10,
 }
 
+// TODO: What does this do?
 var linkRegex, _ = regexp.Compile("<([^|]+)(\\|([^>]+))?>")
```

Comment:
Helps us find and convert [Slack links](https://api.slack.com/docs/message-formatting#linking_to_urls)